scripts/strip-notebook-outputs.py

```
@@ -0,0 +1,21 @@
+
+
+def main() -> None:
+    """Run nbstripout over all notebooks known to the repository index."""
+    result = subprocess.run(
+        ["git", "ls-files", "-z", "--", "*.ipynb"],
+        check=True,
+        capture_output=True,
+    )
+    notebooks = [path.decode() for path in result.stdout.split(b"\0") if path]
+    if notebooks:
+        subprocess.run(["nbstripout", *notebooks], check=True)
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)